src/scripts/benefit-indicator.ts

```diff
@@ -32,6 +32,12 @@ async function main() {
 
     const { api } = await getApp({ baseUrl: opts.url });
     const metadata = await getMetadata(api);
+
+    const defaultCategoryCombo = metadata.categoryCombos.find(
+        categoryCombo => categoryCombo.name === DEFAULT_CATEGORY_COMBO_NAME
```

Comment:
[no need to change] typically we use the code, but the name is always default also.